### DISCUSSION TITLE
Increase auditor user permissions

### DISCUSSION THREAD

**Discussion by Rocky-Jinxing** [rustdesk/rustdesk-server-pro#256] [Q&A]
The Pro version of the self-built server adds auditor user rights. Now it only has ordinary user and administrator rights. When the control background is turned on to disable new devices, ordinary users want to enable the added new controlled terminals. They do not have permission to enable it after entering the background, but The administrator has too much authority and can see all functions, including keys, which will be risky. So I hope to add an auditor's permissions between ordinary users and administrators. The auditor can be responsible for enabling and disabling devices.

**Comment by rustdesk** [comment]
Agree, we will investigate and give a solution.

**Comment by rustdesk** [comment]
Close since transfered to issue
